src/jsx/ChatBox/test_message.jsx

```diff
@@ -0,0 +1,33 @@
+/** @jsx React.DOM */
+var Message = require("./Message.jsx");
+
+// Mock
+var message = {
+  timestamp: 10,
+  user: "mocked",
+  text: "mocked text"
+};
+
+var message_html = "[10] mocked: mocked text";
```

Comment:
of course if the UI behavior like how it shows a message changes, this test will change.... that's what we're testing.